keel-orca/src/main/kotlin/com/netflix/spinnaker/keel/model/OrcaNotification.kt

```diff
@@ -26,7 +26,8 @@ import com.netflix.spinnaker.keel.model.NotificationEvent.ORCHESTRATION_COMPLETE
 import com.netflix.spinnaker.keel.model.NotificationEvent.ORCHESTRATION_FAILED
 import com.netflix.spinnaker.keel.model.NotificationEvent.ORCHESTRATION_STARTING
 
-data class EchoNotification(
+// This gets translated into an echo notification format in orca
+data class OrcaNotification(
```

Comment:
I renamed this because it is actually an orca-specific format to represent notifications, and not what the echo API uses (orca does the translation).